offline_imu_cam/data_loading.hpp

```diff
@@ -43,31 +43,50 @@ typedef struct {
 
 static
 std::map<ullong, sensor_types>
-load_data(const std::string& data_path) {
+load_data() {
+	const char* illixr_data_c_str = std::getenv("ILLIXR_DATA");
+	if (!illixr_data_c_str) {
+		std::cerr << "Please define ILLIXR_DATA" << std::endl;
+		abort();
+	}
+	std::string illixr_data = std::string{illixr_data_c_str};
+
 	std::map<ullong, sensor_types> data;
 
-	std::ifstream imu0_file {data_path + "imu0/data.csv"};
-	assert(imu0_file.good());
+	const std::string imu0_subpath = "/imu0/data.csv";
+	std::ifstream imu0_file {illixr_data + imu0_subpath};
+	if (!imu0_file.good()) {
+		std::cerr << "${ILLIXR_DATA}" << imu0_subpath << " (" << illixr_data << imu0_subpath << ") is not a good path" << std::endl;
+		abort();
+	}
```

Comment:
Very minor, and not required for this PR, but there is a lot of redundancy in how files are read in the data loaders of both offline_imu_cam and ground_truth_slam. In the future, it would be nice to have a common data loading class.